hawtio-system/src/main/java/io/hawt/web/RedirectFilter.java

```diff
@@ -42,7 +42,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
     }
 
     private void process(HttpServletRequest request, HttpServletResponse response, FilterChain chain) throws IOException, ServletException {
-        String uri = request.getRequestURI();
+        String uri = request.getPathInfo();
```

Comment:
What kind of bug does it fix?  Can you first file an issue and explain what bug you've found before submitting a pull req?